### PR TITLE
New version shining light.open ssl.light version 3.0.5

### PR DESCRIPTION
**Pull request by NSO73**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65450)

### CODE REVIEW

File: manifests/s/ShiningLight/OpenSSL/Light/3.0.5/ShiningLight.OpenSSL.Light.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ShiningLight.OpenSSL.Light
+PackageVersion: 3.0.5
```
Comment (Trenly) [inline manifests/s/ShiningLight/OpenSSL/Light/3.0.5/ShiningLight.OpenSSL.Light.installer.yaml:4]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/s/ShiningLight/OpenSSL/Light/3.0.5/ShiningLight.OpenSSL.Light.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: ShiningLight.OpenSSL.Light
+PackageVersion: 3.0.5
```
Comment (Trenly) [inline manifests/s/ShiningLight/OpenSSL/Light/3.0.5/ShiningLight.OpenSSL.Light.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/s/ShiningLight/OpenSSL/Light/3.0.5/ShiningLight.OpenSSL.Light.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: ShiningLight.OpenSSL.Light
+PackageVersion: 3.0.5
```
Comment (Trenly) [inline manifests/s/ShiningLight/OpenSSL/Light/3.0.5/ShiningLight.OpenSSL.Light.yaml:4]:
```suggestion
PackageVersion: 1.0.0
```

**Review by Trenly — COMMENTED** [review]
The PackageVersion must match control panel; When updating this, you will need to update the name of the version folder also.

**Review by Trenly — CHANGES_REQUESTED** [review]
See above

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**